testdata/baselines/reference/submodule/conformance/templateLiteralTypesPatterns.errors.txt.diff

```diff
@@ -77,15 +59,3 @@
      
      declare function numbers(x: `${number}`): void;
      // the following should work
-@@= skipped -133, +133 lines =@@
-     
-     // whitespace and comments aren't part of numbers
-     bigints("- 1");
--            ~~~~~
--!!! error TS2345: Argument of type '"- 1"' is not assignable to parameter of type '`${bigint}`'.
-     bigints("-/**/1");
--            ~~~~~~~~
--!!! error TS2345: Argument of type '"-/**/1"' is not assignable to parameter of type '`${bigint}`'.
```

Comment:
This is the case we're fixing; skipTrivial=false is required here to make the scanner return a space or comment token before the bigint.